### PR TITLE
Define Django Data Models and Register Them

### PR DESCRIPTION
**Pull request by danie1zhang**
This PR modifies `backend/settings.py` and creates new files `backend/models.py` and `backend/admin.py` to define Django Data Models and register them. So far, this PR has defined the data models in `backend/models.py`, added `backend` to `INSTALLED_APPS` in `backend/settings.py`, and registered these data models in `backend/admin.py`. This PR has not been able to migrate the new app in `INSTALLED_APPS` because the docker has some problems related to mysql. It attempts to address issue #45.

### CODE REVIEW

**Comment by jonespm** [comment]
Thanks, you can assign me as a review on these pull requests.

File: backend/admin.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+from canvas-app-explorer.backend.models import LtiTool, CanvasPlacement
```
Comment (jonespm) [inline backend/admin.py:2]: This import should just be

```suggestion
from backend.models import LtiTool, CanvasPlacement

```

**Comment by jonespm** [comment]
Hi Daniel,

I added another commit to this, moving everything into the canvas_app_explorer app created with the start command. I also fixed up manage so it adds this. 

I also ran
`docker exec -it canvas_app_explorer ./manage.py makemigrations`

To create the migrations file. I'm seeing everything working okay. There's a new error with a CSS problem with admin site, but we can fix that separately.

**Comment by jonespm** [comment]
I'm reviewing the model. The only thing I'd change is fix the mispelling of 

`canvas_placmenet`

And set that as a ForeignKey.

**Review by jonespm — APPROVED** [review]

**Comment by danie1zhang** [comment]
These changes look good to me, I am going to squash and merge.

**Pull request merged**